src/main/scala/com/github/conduktor/ksm/source/BitbucketCloudSourceAcl.scala

```diff
@@ -42,7 +44,7 @@ class BitbucketCloudSourceAcl extends SourceAcl {
     password = config.getString(AUTH_PASSWORD_CONFIG)
   }
 
-  override def refresh(): Option[Reader] = {
+  override def refresh(): Option[(AclParser, Reader)] = {
```

Comment:
Would you mind creating a case class to replace this tuple?
It would make the code cleaner in my opinion